LayoutTests/fast/scrolling/programmatic-scroll-merge-position-and-delta-expected.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Only the green part of the div should be visible</title>
+    <style>
+        body {
+            height: 2000px;
+            background-image: repeating-linear-gradient(white, silver 200px);
+            margin: 0px;
+        }
+        
+        .box {
+            width: 200px;
+            height: 200px;
+            background-image: linear-gradient(0deg, green 50%, red 50%);
+        }
+    </style>
+    <script src="../../resources/ui-helper.js"></script>
+    <script>
+        if (window.testRunner)
+            testRunner.waitUntilDone();
+
+        async function doTest()
+        {
+            window.scrollTo(0, 50);
+            window.scrollBy(0, 50);
```

Comment:
I think you have the test and expected files around the wrong way.

Reply:
Fixed.